### ISSUE TITLE
INSERT INTO produces to the wrong topic

### ISSUE THREAD

**Issue by lonsdaleite** [confluentinc/ksql#1790]
"INSERT INTO stream_name ..." command produces messages to STREAM_NAME topic, not the topic under stream "stream_name".

Example:
```
create stream test_stream (id string, f1 string) with (kafka_topic='test-topic', value_format='avro', key='id');
create stream test_stream_2 (id string, f1 string) with (kafka_topic='test-topic-2', value_format='avro', key='id');
insert into test_stream select id, f1 from test_stream_2;
```
The last command produces messages to "TEST_STREAM" topic, but it should produce them to "test-topic" topic.

**Comment by apurvam** [comment]
I think this is a dup of https://github.com/confluentinc/ksql/issues/1711 and is fixed by https://github.com/confluentinc/ksql/pull/1713

**Comment by apurvam** [comment]
It should be fixed in KSQL 5.0.1, release date still pending.
